src/main/java/core/basesyntax/Quadrangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public abstract class Quadrangle extends Figure implements Area {
+    private final int width;
+    private final int height;
+
+    public Quadrangle(String color, int width, int height) {
+        super(color);
+        this.width = width;
+        this.height = height;
+    }
+
+    @Override
+    public double area() {
```

Comment:
The method name `area` should be a verb or a verb phrase to indicate an action, as it's a method that performs a calculation. Consider renaming it to something like `calculateArea`.